packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2386,14 +2385,14 @@ test('director notifiers', async t => {
     'Chit',
     defaultParamValues(chit.brand),
   );
-  state = await E(metrics).getUpdateSince(state.updateCount);
-  t.deepEqual(state.value, {
-    collaterals: [aethBrand, chit.brand],
-    rewardPoolAllocation: {},
+  await m.assertChange({
+    collaterals: { 1: chit.brand },
```

Comment:
since this is expresses changes, it has to reference the place in the array (`1:`)